app/src/main/java/br/ufpe/cin/pet/geoquest/RakingRequest.java

```diff
@@ -0,0 +1,89 @@
+package br.ufpe.cin.pet.geoquest;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.protocol.HTTP;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import br.ufpe.cin.pet.geoquest.classes.Raking;
+
+/**
+ * Created by rbb3 on 06/06/16.
+ */
+public class RakingRequest extends AsyncTask<String, Void, List<Raking> > {
+
+    private final String url;
+
+    private HttpResponse response;
+
+    public RakingRequest(String url) {
+        this.url = url;
+    }
+
+    @Override
+    protected List<Raking> doInBackground(String... params) {
```

Comment:
Eu tinha pensado antes em fazer um método default pra fazer todos requests, onde a gente passa a url como parametro e retorna sempre uma String, que seria o que tu chamou de sresponse. Isso só pra facilitar pra todo mundo, aí o q cada um teria q fazer é só o parse (colocar num JSONObject ou JSONArray...). Isso é possível? To perguntando isso pq n sei se pra fazer o parse pra JSON precisa ser assíncrono tb. Só queria evitar repetição de código.